firebase-perf/src/main/java/com/google/firebase/perf/config/RemoteConfigManager.java

```diff
@@ -48,9 +49,13 @@
   private static final long TIME_AFTER_WHICH_A_FETCH_IS_CONSIDERED_STALE_MS =
       TimeUnit.HOURS.toMillis(12);
   private static final long FETCH_NEVER_HAPPENED_TIMESTAMP_MS = 0;
+  private static final long MIN_APP_START_CONFIG_FETCH_DELAY_MS = 5000;
+  private static final int RANDOM_APP_START_CONFIG_FETCH_DELAY_MS = 25000;
 
   private final ConcurrentHashMap<String, FirebaseRemoteConfigValue> allRcConfigMap;
   private final Executor executor;
+  private final long appStartTime;
+  private final long appStartConfigFetchDelay;
```

Comment:
Can we suffix this with the cardinality of the variable? Eg. appStartConfigFetchDelayInMs? This would make the code more readable and makes it easy to understand for anyone working with the code in the future.

Reply:
done